src/main/java/de/adesso/kicker/user/controller/UserController.java

```diff
@@ -35,4 +39,13 @@ private ModelAndView defaultProfileView(ModelAndView modelAndView, User user) {
         modelAndView.setViewName("sites/profile.html");
         return modelAndView;
     }
+
+    @GetMapping("/js/{id}")
```

Comment:
I believe it would be better to put the JS templates into their own controller and normal mappings.
We have most of our scripts in /js and it would be nicer to have them all there.

Reply:
I'd like to discuss this is a new issue with the team.